src/d3a/models/strategy/area_agents/one_sided_agent.py

```diff
@@ -68,5 +68,9 @@ def event_offer_split(self, *, market_id,  original_offer, accepted_offer, resid
                                      accepted_offer=accepted_offer,
                                      residual_offer=residual_offer)
 
+    def event_offer(self, *, market_id, offer):
+        for engine in sorted(self.engines, key=lambda _: random()):
+            engine.event_offer(market_id=market_id, offer=offer)
+
```

Comment:
Hmm, i am confused. We never dispatched the offer event to the engines before?

Reply:
We used to send the offer event, and trigger automatic offer forwarding whenever the offer event was received. However we removed that some time ago.